src/pages/landing/weekend-trading1/index.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+import Loadable from '@loadable/component'
+import Hero from './components/_hero'
+import ImageTextSwitching from './components/_image-text-switching'
+import Titlebtn from './components/_title-btn'
+import { localize, Localize, WithIntl } from 'components/localization'
+import Layout from 'components/layout/layout'
+import { SEO } from 'components/containers'
+const IconTextRow = Loadable(() => import('./components/_icon-text-row'))
+
+const DP2P_CONTENT = [
+    {
+        title: <Localize translate_text="Synthetic indices" />,
+        subtitle: (
+            <Localize
+                translate_text="Deriv's synthetic indices mimic real-world market movements and offer various volatility levels to suit your risk appetite.<0 /><0 /> Available to trade on Deriv MT5 and Deriv X (with margin), DTrader (with options and multipliers), and Deriv GO (with multipliers)."
+                components={[<br key={0} />]}
+            />
+        ),
+        subtitle_mobile: (
+            <Localize
+                translate_text="Deriv's synthetic indices mimic real-world market movements and offer various volatility levels to suit your risk appetite.<0 /><0 /> Available to trade on Deriv MT5 and Deriv X (with margin), DTrader (with options and multipliers), and Deriv GO (with multipliers)."
+                components={[<br key={0} />]}
+            />
+        ),
+
+        image_name: 'buy_sell',
+        image_alt: localize('Buy and sell'),
+    },
+    {
+        title: <Localize translate_text="Cryptocurrency" />,
+        subtitle: (
+            <Localize
+                translate_text="Correctly predict the price movements of the world's most popular cryptocurrencies. Over 17 crypto pairs to trade on.<0 /><0 />Available to trade on Deriv MT5 and Deriv X (with margin) and DTrader (with multipliers)."
+                components={[<br key={0} />]}
+            />
+        ),
+        subtitle_mobile: (
+            <Localize
+                translate_text="Correctly predict the price movements of the world's most popular cryptocurrencies. Over 17 crypto pairs to trade on.<0 /><0 />Available to trade on Deriv MT5 and Deriv X (with margin) and DTrader (with multipliers)."
+                components={[<br key={0} />]}
+            />
+        ),
+
+        image_name: 'local_currency',
+        image_alt: localize('Local currency'),
+    },
+]
+
+const DP2PHome = () => {
```

Comment:
Can we change the name of this `component` into closer details to `weekend-trading`

Reply:
done